bootstrap/cmd/kfctl/cmd/set-image-name.go

```diff
@@ -63,6 +64,13 @@ The flatten flag discards both registry and name components except for the last
 			return err
 		}
 
+		changeList := []string{}
```

Comment:
+1

Reply:
Makes sense.
My original thought was that printing it at the end would separate it from the logging text and make it more legible if the user doesn't pipe stdout to somewhere else.

Reply:
if the user wants to use the output, we should only print the (old?) image names, nothing else. This is the case without the verbose flag if I'm not mistaken. Maybe try to work out the bash commands you would use to see how the output would suit you best.

Reply:
sorry, my wrong. As you pointed it is better to print at the end. But print some with message will help to understand better. 
I meant print some message before printing changelist

Reply:
I documented the functionality in the command's doc.
I don't think it's necessary to print a message before the changelist because:
Changes have been logged to stderr, the changelist is a machine-readable representation of them that the user can then pipe into another program to eg mirror the images.